### ISSUE TITLE
className styles are purged if `purge` object is present in tailwind.config.js

### ISSUE THREAD

**Issue by codemonkeynorth** [anantoghosh/gatsby-plugin-purgecss#925]
**Describe the bug**
since purgecss is now included in tailwind, `className` styles appear to be purged if `purge` object is present in `tailwind.config.js` even if the `purge` object is empty `{}`

**To Reproduce**
```
"gatsby": "^2.24.79",
"gatsby-plugin-postcss": "^3.0.3",
"gatsby-plugin-purgecss": "^5.0.0",
"tailwindcss": "^1.9.4"
```

**Expected behavior**
tailwind styles in `className` should not be purged

**Additional information**
This may just be a case of adding documentation to ensure the `purge: {}` object is not present on the tailwind config when using `gatsby-plugin-purgecss`

Tailwind does actually give a warning suggesting this
```
warn - Tailwind is not purging unused styles because no template paths have been provided.
warn - If you have manually configured PurgeCSS outside of Tailwind or are deliberately not removing unused styles, set `purge: false` in your Tailwind config
warn - https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
```

basically set `purge: false` in `tailwind.config.js` if using `gatsby-plugin-purgecss` should fix it I think
